web/frontend/src/components/slam-2d-render.vue

```diff
@@ -350,12 +282,16 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
     points,
     intersectionPlane,
     axesPos,
-    axesNeg
+    axesNeg,
+    baseMarker,
+    destMarker
   );
 
   if (props.pose !== undefined) {
     updatePose(props.pose!);
```

Comment:
Why is this still updating the pose?

Reply:
This is unchanged from main, I'm leaving any further optimizations for followup